src/AltinnCore/Designer/Views/Model/Index.cshtml

```diff
@@ -83,7 +83,7 @@
     </div>
 
     <div class="tab-pane fade in" id="json" role="tabpanel"> 
-        <a title="Last ned JSON-modell" asp-action="GetJson" asp-controller="Model" asp-route-org="@org" asp-route-app="@service" download="@service-model.json">
+        <a title="Last ned JSON-modell" asp-action="GetJson" asp-controller="Model" asp-route-org="@org" asp-route-app="@app" download="service-model.json">
```

Comment:
why is the "@" missing in download?

Reply:
This was a misunderstanding, I have now changed to @app-model.json, same on the others.